build/Common.prod.props

```diff
@@ -7,9 +7,9 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
 
-    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Condition=" $(OS) == 'Windows_NT'">
+    <!-- <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Condition=" $(OS) == 'Windows_NT'">
```

Comment:
@open-telemetry/dotnet-approvers and @open-telemetry/dotnet-maintainers: 
Please help in reviewing the TFMs. Once I get a good idea of people's take of the current shape of the TFMs, I will enable the API check back. Thank you.

Reply:
This [set ](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4595/files#diff-c123fcece83dd5f1a59e23928148519fe1d6515f9c14f1337c7a6878a2720911R47)of TFMs look good to me.